src/ui/zcl_abapgit_gui_page_repo_over.clas.abap

```diff
@@ -600,4 +599,4 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_OVER IMPLEMENTATION.
     <ls_col>-add_tz = iv_add_tz.
     <ls_col>-allow_order_by = iv_allow_order_by.
   ENDMETHOD.
-ENDCLASS.
+ENDCLASS.
```

Comment:
I get a empty diff in abapGit after pulling, due to this whitespace change?

Reply:
fixed, tried pulling and there's no diff